ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -301,9 +300,15 @@ Loader {
                 return StatusMessage.ContentType.Unknown;
             }
         }
-    }
-
 
+        function addReactionClicked(signalSender) {
+            if (root.isChatBlocked)
+                return
+            if (d.addReactionAllowed)
+                return
```

Comment:
I don't get this line. If adding reactions is allowed, we don't show the context menu?

Reply:
Will drop explicit `addReactionAllowed` check. It turned out that `isChatBlocked` already does required checks:
https://github.com/status-im/status-desktop/blob/ca1a90154e51a9233f71ba808902658f3838858e/ui/app/AppLayouts/Chat/views/ChatContentView.qml#L112